frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeSection.tsx

```diff
@@ -88,23 +86,19 @@ const ServingRuntimeSizeSection: React.FC<ServingRuntimeSizeSectionProps> = ({
       >
         <Stack hasGutter>
           <StackItem>
-            <Select
-              id="model-server-size-selection"
-              isOpen={sizeDropdownOpen}
-              placeholderText="Select a model server size"
-              onToggle={(e, open) => setSizeDropdownOpen(open)}
+            <SimpleSelect
```

Comment:
When i first tab to this select and press enter, i cannot use the arrow keys to change options.
But after clicking with the mouse an option, the same steps above work to use arrow keys to change options.

Reply:
fixed